src/index.js

```diff
@@ -20,22 +20,27 @@ class MapView extends Component {
   };
 
   render() {
-    if (!this.state.center)
+    if (!this.props.region)
       return (
         <View style={styles.container}>
           <ActivityIndicator />
         </View>
       );
+    const center = { 
```

Comment:
The component should be usable with or without a region prop

Reply:
Fair enough, will fix...